profiler/quickstart/main.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An example of starting https://cloud.google.com/profiler."""
+
+# [START profiler_python_quickstart]
+import googlecloudprofiler
+
+
+def main():
+    # Profiler initialization. It starts a daemon thread which continuously
+    # collects and uploads profiles. Best done as early as possible.
+    try:
+        googlecloudprofiler.start(
+            service='hello-profiler',
+            service_version='1.0.1',
+            verbose=3,
+            # project_id must be set if not running on GCP.
+            # project_id='my-project-id',
+        )
+    except (ValueError, NotImplementedError) as e:
+        print(e)  # Handles errors here
```

Comment:
Can you provide some explanation for when ValueError or NotImplementedError might occur?

Reply:
It's a little long and it will be presented in the doc surrounding this code snippet. I'm a little hesitate to make this code snippet too long.